api/base/filters.py

```diff
@@ -66,6 +69,33 @@ def filter_queryset(self, request, queryset, view):
             return queryset.sort(*ordering)
         return queryset
 
+    def check_serializer_fields(self, queryset, query_params, view, ordering):
+        serializer_class = view.serializer_class
```

Comment:
I'd get the `serializer_class `differently because it's not always hardcoded. 

 I'm looking at `OrderingFilter.get_default_valid_fields` which is called as part of `self.get_ordering(request, queryset, view)`, above:
```python
 if hasattr(view, 'get_serializer_class'):
            try:
                serializer_class = view.get_serializer_class()
            except AssertionError:
                # Raised by the default implementation if
                # no serializer_class was found
                serializer_class = None
        else:
            serializer_class = getattr(view, 'serializer_class', None)
```

Not all views have `serializer_class` hardcoded. Some, the serializer_class is determined based on the request method in `view.get_serializer_class()`